app/controllers/home_controller.rb

```diff
@@ -0,0 +1,9 @@
+class HomeController < ApplicationController
+  layout "home"
+
+  def index; end
+
+  def about_us; end
```

Comment:
remove `about_us` and `contact_us` for now. Let's just have a single page `home`